joblib/numpy_pickle.py

```diff
@@ -270,6 +273,28 @@ def save(self, obj):
                         traceback.format_exc()))
         return Pickler.save(self, obj)
 
+    def save_bytes(self, obj):
+        """Strongly inspired from python 2.7 pickle.Pickle.save_bytes"""
+        if self.bin:
+            n = len(obj)
+            if n < 256:
+                self.write(pickle.SHORT_BINSTRING + asbytes(chr(n)) + obj)
+            else:
+                self.write(pickle.BINSTRING + struct.pack("<i", n) + obj)
```

Comment:
I am more than happy to follow your suggestion, but FWIW these lines come from the python 2.7 source code [here](https://github.com/python/cpython/blob/2.7/Lib/pickle.py#L480).